server/db/models/GameTable.js

```diff
@@ -0,0 +1,11 @@
+const Sequelize = require('sequelize')
+const db = require('../db')
+
+const GameTable = db.define('gameTable', {
+accessCode: {
+    type: Sequelize.STRING,
```

Comment:
Maybe UUID, and validate unique?

Reply:
It looks like UUIDs are 32 characters long and follow a specific format, and since this is our access code, it should probably just be like a random 6- or 8-character string. Validate unique though I am adding in.